src/mapeo-manager.js

```diff
@@ -263,19 +288,25 @@ export class MapeoManager {
       projectId
     )
 
-    const project = new MapeoProject({
+    const project = this.#getProjectInstance(projectKeys)
+
+    // 3. Keep track of project instance as we know it's a properly existing project
+    this.#activeProjects.set(projectPublicId, project)
+
+    return project
+  }
+
+  /** @param {ProjectKeys} projectKeys */
+  #getProjectInstance(projectKeys) {
```

Comment:
nit: this seems more like a create than a get

Reply:
Yes, makes sense